pyramid/config/views.py

```diff
@@ -568,9 +568,8 @@ def add(self, view, order, accept=None, phash=None):
             else:
                 subset.append((order, view, phash))
                 subset.sort(key=operator.itemgetter(0))
-            accepts = set(self.accepts)
-            accepts.add(accept)
-            self.accepts = list(accepts) # dedupe
```

Comment:
There is no concept of "default" accept, and if you use `.scan()` across multiple files for example, it becomes even murkier because it highly depends on which one gets scanned first as to which one becomes the "default". This is a known issue.